package.json

```diff
@@ -23,7 +23,9 @@
     "test": "react-scripts test --passWithNoTests",
     "eject": "react-scripts eject",
     "lintfix": "eslint ./src --fix",
-    "deploy-test": "npm run build && firebase deploy --only hosting:eden-test",
+    "build-test": "PUBLIC_URL=https://eden-test.firebaseapp.com/ npm run build",
+    "build-prod": "PUBLIC_URL=https://eden-5d453.firebaseapp.com/  npm run build",
+    "deploy-test": "npm run build-test && firebase deploy --only hosting:eden-test",
     "deploy-prod": "npm run build && firebase deploy --only hosting:eden-5d453"
```

Comment:
Shouldnt it be "npm run build-prod"  instead of "npm run build" for deploy-prod??